generator/list.tmpl

```diff
@@ -9,7 +9,7 @@ depending on their needs and workflow.
 
 Each group's material is in its subdirectory in this project.
 
-When the need arises, a [new SIG can be created](sig-creation-procedure.md)
+When the need arises, a [new SIG can be created](sig-governance.md#sig-creation-and-maintenance-procedure)
```

Comment:
@pierreozoux thanks! you'll need to run `make generate` for the tests to pass :)

Reply:
can't this `make generate` be part of a pipeline? I'd love to help you build this pipeline, but I don't have the time at the moement :/

Reply:
@pierreozoux The sig-list.md file is intended to be tracked in git and not an artifact stored some place else, it does make sense to run it before committing.

I completely forgot about running `make generate` to ensure the synced changes when updating the template. My apologies. I should have mentioned that in the other issue. >_<